pkg/webui/console/views/application-payload-formatters/index.js

```diff
@@ -41,6 +41,8 @@ import {
 import style from './application-payload-formatters.styl'
 
 const m = defineMessages({
+  infoText:
+    'Application payload formatters are executed on all devices in the application, unless the user defines a device-specific payload formatter.',
```

Comment:
These payload formatters are executed on uplink messages from all devices in this application. If payload formatters are also set for a specific device, only those will be executed for uplinks from that device.